lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -135,12 +137,12 @@ finalizeCredit bilateralCredit = do
                 commit conn
 
 
-createPendingRecord :: Pool Connection -> CreditRecord -> LndrHandler ()
-createPendingRecord pool signedRecord = do
+createPendingRecord :: Integer -> Pool Connection -> CreditRecord -> LndrHandler ()
```

Comment:
I don't think adding a `nonceIncrement` parameter should be necessary. Why did you do this?